packages/lit/src/index.all.ts

```diff
@@ -36,6 +36,7 @@ export {
   svg as staticSvg,
   unsafeStatic,
   withStatic,
+  StaticValue,
```

Comment:
Despite what the comment says, I wonder if `StaticValue` actually needs to be added here since this file is used just to create the JS bundle. The TS compilation step would strip it away anyway so it probably doesn't matter either way.

Reply:
If it's not doing anything useful, I'll just remove it and update the changelist.